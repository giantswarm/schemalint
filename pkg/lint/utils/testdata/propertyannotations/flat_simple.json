{
  "$id": "https://example.com/person.schema.json",
  "properties": {
    "lastName": {
      "type": "string",
      "title": "Last Name"
    }
  },
  "type": "object"
}
