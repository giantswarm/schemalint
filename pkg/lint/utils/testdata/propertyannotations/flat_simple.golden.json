{
  "/lastName": {
    "Title": {
      "Title": "Last Name",
      "ReferenceLevel": 0
    }
  }
}
